pyproject.toml

```diff
@@ -1,11 +1,11 @@
 [build-system]
-requires = ["setuptools", "wheel", "setuptools_scm"]
+requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=4.1.0"]
```

Comment:
@chrisjbillington do you think `setuptools_scm` can go from here?

Reply:
No, I think it's necessary - all build-time dependencies need to be specified there.

Though I don't know why the [toml] extras bit is there, would not think that is needed.

Reply:
The configuration might be moved to setup.py, but it needs to be specified in pyproject.toml/build_system/requires to ensure the package is actually installed before setup.py runs, when you're doing a pep517 build.

Reply:
Removed `[toml]` from this build requirement as the options are not specified there (see labscript-suite/labscript-utils#73 for justification).

Reply:
Aaah, that's why it was specified - so that setuptools_scm can read its own config from pyproject.toml. Kind of confusing since obviously the toml module already exists for this file to be read in the first place - but it might be a vendored module within another package, or not exist in the isolated build environment that setup.py is run in, so still needs to be installed.

But yes. Not if the config is not in pyproject.toml anymore.